cypress/integration/shared/common.ts

```diff
@@ -78,3 +79,103 @@ export const waitForClusterState = (cy: Cypress.cy, desiredState: string, retrie
     }
   });
 };
+
+export const setProxyValues = (httpProxy = null, httpsProxy = null, noProxy = null) => {
```

Comment:
Since this is related to a specific page (Cluster configuration in this case), then it should not be in the `common.ts` file

Reply:
setProxyValues is required by generateIso, and they are both in common.ts. It would not be proper to put it in clusterConfiguration (almost everything is on that page).